mobile/apps/artwork/components/meta_data/templates/partner.jade

```diff
@@ -3,8 +3,8 @@
     //- Drop Header
   else
     h2.artwork-header= artwork.is_for_sale ? 'Offered by' : artwork.partner.type.replace('Institutional Seller', 'Institution')
-  if artwork.partner.type == 'Auction'
-    a.artwork-partner-link.chevron-arrow-button( href='/auction' + artwork.partner.href )= artwork.partner.name
+  if artwork.partner.type == 'Auction' || artwork.partner.type == 'Auction House'
```

Comment:
I left the original check for `'Auction'` to avoid a regression, but we actually transform gravity's `'Auction'` partner type to `'Auction House'` in [metaphysics](https://github.com/artsy/metaphysics/blob/24a7d68717b000830299b4531da00c68ccfe40b5/schema/partner.js#L152) (not entirely sure why).